imsv-docs-astro/src/content/docs/guides/authentication/authentication.mdoc

```diff
@@ -0,0 +1,192 @@
+---
+title: Authentication
+slug: guides/authentication
+description: |
+  How to authenticate with Immersve APIs.
+sidebar:
+  order: 1
+---
+
+Immersve APIs are authenticated with either long-lived API Keys or short-lived
+User Session access tokens. Permissions are controlled by roles, scopes and
```

Comment:
Could link 'roles' to 'Roles and Permissions' anchor and 'scopes' to 'Client Scopes' anchor, might make the paragraph too 'linky' though.

Reply:
I deliberately left this as non-hyperlinked because internal links are hard to maintain.